Generating model with property of object type doesn't seem to work

### Tell us the extension you're using

Client

### I tried this:

I'm using this openapi file: 

[docling.json](https://github.com/user-attachments/files/20696586/docling.json)

and running the generator on it.

### This happened:


Specifically for the `PictureDescriptionLocal` model element, it has this property:

```json
"generation_config": {
   "type": "object",
  "title": "Generation Config",
  "description": "Config from https://huggingface.co/docs/transformers/en/main_classes/text_generation#transformers.GenerationConfig",
  "default": {
    "max_new_tokens": 200,
    "do_sample": false
  },
  "examples": [
    {
      "do_sample": false,
      "max_new_tokens": 200
    }
  ]
}
```

Which ends up generating a Java class that has something like this in it:

```java
private Object generationConfig = {max_new_tokens=200, do_sample=false};
```

which obviously doesn't compile. The openapi document isn't mine, so I can't change it. Is there a reason why this might be generating incorrectly?

Here's the whole output fileL

```java
package io.quarkiverse.docling.runtime.client.model;


import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonTypeName;
import com.fasterxml.jackson.annotation.JsonValue;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import com.fasterxml.jackson.annotation.JsonProperty;

@com.fasterxml.jackson.annotation.JsonIgnoreProperties(ignoreUnknown = true)
public class PictureDescriptionLocal  {

    /**
      * Repository id from the Hugging Face Hub.
     **/
    private String repoId;
    /**
      * Prompt used when calling the vision-language model.
     **/
    private String prompt = "Describe this image in a few sentences.";
    /**
      * Config from https://huggingface.co/docs/transformers/en/main_classes/text_generation#transformers.GenerationConfig
     **/
    private Object generationConfig = {max_new_tokens=200, do_sample=false};

    /**
    * Repository id from the Hugging Face Hub.
    * @return repoId
    **/
    @JsonProperty("repo_id")
    public String getRepoId() {
        return repoId;
    }

    /**
     * Set repoId
     **/
    public void setRepoId(String repoId) {
        this.repoId = repoId;
    }

    public PictureDescriptionLocal repoId(String repoId) {
        this.repoId = repoId;
        return this;
    }

    /**
    * Prompt used when calling the vision-language model.
    * @return prompt
    **/
    @JsonProperty("prompt")
    @com.fasterxml.jackson.annotation.JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)
    public String getPrompt() {
        return prompt;
    }

    /**
     * Set prompt
     **/
    public void setPrompt(String prompt) {
        this.prompt = prompt;
    }

    public PictureDescriptionLocal prompt(String prompt) {
        this.prompt = prompt;
        return this;
    }

    /**
    * Config from https://huggingface.co/docs/transformers/en/main_classes/text_generation#transformers.GenerationConfig
    * @return generationConfig
    **/
    @JsonProperty("generation_config")
    @com.fasterxml.jackson.annotation.JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)
    public Object getGenerationConfig() {
        return generationConfig;
    }

    /**
     * Set generationConfig
     **/
    public void setGenerationConfig(Object generationConfig) {
        this.generationConfig = generationConfig;
    }

    public PictureDescriptionLocal generationConfig(Object generationConfig) {
        this.generationConfig = generationConfig;
        return this;
    }

    /**
     * Create a string representation of this pojo.
     **/
    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("class PictureDescriptionLocal {\n");

        sb.append("    repoId: ").append(toIndentedString(repoId)).append("\n");
        sb.append("    prompt: ").append(toIndentedString(prompt)).append("\n");
        sb.append("    generationConfig: ").append(toIndentedString(generationConfig)).append("\n");
        
        sb.append("}");
        return sb.toString();
    }

    /**
     * Compares this object to the specified object. The result is
     * {@code true} if and only if the argument is not
     * {@code null} and is a {@code PictureDescriptionLocal} object that
     * contains the same values as this object.
     *
     * @param   obj   the object to compare with.
     * @return  {@code true} if the objects are the same;
     *          {@code false} otherwise.
     **/
    @Override
    public boolean equals(Object obj) {
        if (this == obj) return true;
        if (obj == null || getClass() != obj.getClass()) return false;

        PictureDescriptionLocal model = (PictureDescriptionLocal) obj;

        return java.util.Objects.equals(repoId, model.repoId) &&
        java.util.Objects.equals(prompt, model.prompt) &&
        java.util.Objects.equals(generationConfig, model.generationConfig);
    }

    /**
     * Returns a hash code for a {@code PictureDescriptionLocal}.
     *
     * @return a hash code value for a {@code PictureDescriptionLocal}.
     **/
    @Override
    public int hashCode() {
        return java.util.Objects.hash(repoId,
        prompt,
        generationConfig);
    }

    /**
     * Convert the given object to string with each line indented by 4 spaces
     * (except the first line).
     */
    private static String toIndentedString(Object o) {
        if (o == null) {
            return "null";
        }
        return o.toString().replace("\n", "\n    ");
    }

    @com.fasterxml.jackson.annotation.JsonIgnoreProperties(ignoreUnknown = true)
    public static class PictureDescriptionLocalQueryParam  {

        @jakarta.ws.rs.QueryParam("repoId")
        private String repoId;
        @jakarta.ws.rs.QueryParam("prompt")
        private String prompt = "Describe this image in a few sentences.";
        @jakarta.ws.rs.QueryParam("generationConfig")
        private Object generationConfig = {max_new_tokens=200, do_sample=false};

        /**
        * Repository id from the Hugging Face Hub.
        * @return repoId
        **/
        @com.fasterxml.jackson.annotation.JsonProperty("repo_id")
        public String getRepoId() {
            return repoId;
        }

        /**
         * Set repoId
         **/
        public void setRepoId(String repoId) {
            this.repoId = repoId;
        }

        public PictureDescriptionLocalQueryParam repoId(String repoId) {
            this.repoId = repoId;
            return this;
        }

        /**
        * Prompt used when calling the vision-language model.
        * @return prompt
        **/
        @com.fasterxml.jackson.annotation.JsonProperty("prompt")
        public String getPrompt() {
            return prompt;
        }

        /**
         * Set prompt
         **/
        public void setPrompt(String prompt) {
            this.prompt = prompt;
        }

        public PictureDescriptionLocalQueryParam prompt(String prompt) {
            this.prompt = prompt;
            return this;
        }

        /**
        * Config from https://huggingface.co/docs/transformers/en/main_classes/text_generation#transformers.GenerationConfig
        * @return generationConfig
        **/
        @com.fasterxml.jackson.annotation.JsonProperty("generation_config")
        public Object getGenerationConfig() {
            return generationConfig;
        }

        /**
         * Set generationConfig
         **/
        public void setGenerationConfig(Object generationConfig) {
            this.generationConfig = generationConfig;
        }

        public PictureDescriptionLocalQueryParam generationConfig(Object generationConfig) {
            this.generationConfig = generationConfig;
            return this;
        }

        /**
         * Create a string representation of this pojo.
         **/
        @Override
        public String toString() {
            StringBuilder sb = new StringBuilder();
            sb.append("class PictureDescriptionLocalQueryParam {\n");

            sb.append("    repoId: ").append(toIndentedString(repoId)).append("\n");
            sb.append("    prompt: ").append(toIndentedString(prompt)).append("\n");
            sb.append("    generationConfig: ").append(toIndentedString(generationConfig)).append("\n");
            sb.append("}");
            return sb.toString();
        }

        /**
         * Convert the given object to string with each line indented by 4 spaces
         * (except the first line).
         */
        private static String toIndentedString(Object o) {
            if (o == null) {
                return "null";
            }
            return o.toString().replace("\n", "\n    ");
        }
    }}
```

### I expected this:

I expected the generated Java code to compile, which it does not.

### Is there a workaround?

Not that I've found

### How can we try to reproduce the issue?

1. Clone https://github.com/quarkiverse/quarkus-docling
2. `cd quarkus-docling`
3. `./mvnw -f runtime clean compile`

The offending file that's generated is `target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java`

See the errror:

```
[INFO] --- compiler:3.14.0:compile (default-compile) @ quarkus-docling ---
[INFO] Recompiling the module because of changed source code.
[INFO] Compiling 96 source files with javac [debug deprecation parameters release 17] to target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,39] illegal initializer for java.lang.Object
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,40] cannot find symbol
  symbol:   variable max_new_tokens
  location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,60] cannot find symbol
  symbol:   variable do_sample
  location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,43] illegal initializer for java.lang.Object
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,44] cannot find symbol
  symbol:   variable max_new_tokens
  location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal.PictureDescriptionLocalQueryParam
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,64] cannot find symbol
  symbol:   variable do_sample
  location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal.PictureDescriptionLocalQueryParam
[INFO] 6 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.456 s
[INFO] Finished at: 2025-06-11T14:51:50-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.14.0:compile (default-compile) on project quarkus-docling: Compilation failure: Compilation failure: 
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,39] illegal initializer for java.lang.Object
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,40] cannot find symbol
[ERROR]   symbol:   variable max_new_tokens
[ERROR]   location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[27,60] cannot find symbol
[ERROR]   symbol:   variable do_sample
[ERROR]   location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,43] illegal initializer for java.lang.Object
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,44] cannot find symbol
[ERROR]   symbol:   variable max_new_tokens
[ERROR]   location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal.PictureDescriptionLocalQueryParam
[ERROR] /Users/edeandre/workspaces/quarkus/quarkus-docling/runtime/target/generated-sources/open-api/io/quarkiverse/docling/runtime/client/model/PictureDescriptionLocal.java:[163,64] cannot find symbol
[ERROR]   symbol:   variable do_sample
[ERROR]   location: class io.quarkiverse.docling.runtime.client.model.PictureDescriptionLocal.PictureDescriptionLocalQueryParam
[ERROR] -> [Help 1]
```

### Anything else?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Quarkus OpenApi version or git rev

2.10.0

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### Community Notes

<!-- Please keep this note for the community -->
* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.name: Bug Report 🐞

--- a/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java
+++ b/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java
@@ -473,12 +473,10 @@ public abstract class OpenApiGeneratorCodeGenBase implements CodeGenProvider {
             Class<T> propertyType) {
 
         Optional<String> possibleConfigKey = getConfigKeyValue(config, openApiFilePath);
-        if (possibleConfigKey.isPresent()) {
-            return getValuesByConfigKey(config, CodegenConfig.getSpecConfigNameByConfigKey(possibleConfigKey.get(), configName),
-                    propertyType, configName);
-        }
+        return possibleConfigKey
+                .flatMap(s -> getValuesByConfigKey(config, CodegenConfig.getSpecConfigNameByConfigKey(s, configName),
+                        propertyType, configName));
 
-        return Optional.empty();
     }
 
     private <K, V> Optional<Map<K, V>> getConfigKeyValues(final SmallRyeConfig config, final Path openApiFilePath,
@@ -486,10 +484,7 @@ public abstract class OpenApiGeneratorCodeGenBase implements CodeGenProvider {
             Class<K> kClass, Class<V> vClass) {
 
         Optional<String> possibleConfigKey = getConfigKeyValue(config, openApiFilePath);
-        if (possibleConfigKey.isPresent()) {
-            return getValuesByConfigKey(config, configName, kClass, vClass, possibleConfigKey.get());
-        }
+        return possibleConfigKey.flatMap(s -> getValuesByConfigKey(config, configName, kClass, vClass, s));
 
-        return Optional.empty();
     }
 }

--- a/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorOutputPaths.java
+++ b/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorOutputPaths.java
@@ -2,7 +2,6 @@ package io.quarkiverse.openapi.generator.deployment.codegen;
 
 import java.nio.file.Path;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collection;
 import java.util.Iterator;
 import java.util.List;
@@ -12,7 +11,7 @@ public class OpenApiGeneratorOutputPaths {
     public static final String OPENAPI_PATH = "open-api";
     public static final String STREAM_PATH = "open-api-stream";
 
-    private static final Collection<String> rootPaths = Arrays.asList(STREAM_PATH);
+    private static final Collection<String> rootPaths = List.of(STREAM_PATH);
 
     public static Path getRelativePath(Path path) {
         List<String> paths = new ArrayList<>();

--- a/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java
+++ b/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java
@@ -154,7 +154,7 @@ public class QuarkusJavaClientCodegen extends JavaClientCodegen {
             // Property is a `type: object` without `additionalProperties: true`, without `properties`, but has `default` values set!
             // In this peculiar situation, the template will try to initialize a Java Object with such values, and it will fail to compile.
             // See https://github.com/quarkiverse/quarkus-openapi-generator/issues/1185 for more context.
-            if (p.getType().equals("object") && p.getDefault() != null && p.getAdditionalProperties() == null
+            if ("object".equals(p.getType()) && p.getDefault() != null && p.getAdditionalProperties() == null
                     && p.getItems() == null) {
                 p.setAdditionalProperties(true);
             }

--- a/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java
+++ b/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java
@@ -148,6 +148,20 @@ public class QuarkusJavaClientCodegen extends JavaClientCodegen {
         return codegenModel;
     }
 
+    @Override
+    public CodegenProperty fromProperty(String name, Schema p, boolean required, boolean schemaIsFromAdditionalProperties) {
+        if (p != null && p.getType() != null) {
+            // Property is a `type: object` without `additionalProperties: true`, without `properties`, but has `default` values set!
+            // In this peculiar situation, the template will try to initialize a Java Object with such values, and it will fail to compile.
+            // See https://github.com/quarkiverse/quarkus-openapi-generator/issues/1185 for more context.
+            if (p.getType().equals("object") && p.getDefault() != null && p.getAdditionalProperties() == null
+                    && p.getItems() == null) {
+                p.setAdditionalProperties(true);
+            }
+        }
+        return super.fromProperty(name, p, required, schemaIsFromAdditionalProperties);
+    }
+
     private void warnIfDuplicated(CodegenModel m) {
         Set<String> propertyNames = new TreeSet<>();
         for (CodegenProperty element : m.allVars) {

--- a/client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java
+++ b/client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java
@@ -718,6 +718,30 @@ public class OpenApiClientGeneratorWrapperTest {
         }
     }
 
+    @Test
+    void verifyAllowAdditionalPropertiesIfNotPresent() throws Exception {
+        List<File> generatedFiles = createGeneratorWrapperReactive("issue-1185.json")
+                .generate("org.issue1185")
+                .stream()
+                .filter(file -> file.getPath().endsWith("PictureDescriptionLocal.java")).toList();
+
+        assertThat(generatedFiles).isNotEmpty();
+        // generationConfig
+        for (File file : generatedFiles) {
+            try {
+                Optional<FieldDeclaration> var = StaticJavaParser.parse(file)
+                        .findAll(FieldDeclaration.class).stream()
+                        .filter(c -> c.getVariable(0)
+                                .getNameAsString().equals("generationConfig"))
+                        .findFirst();
+                assertThat(var).isPresent();
+                assertThat(var.get().getElementType().asString()).contains("Map<String,Object>");
+            } catch (FileNotFoundException e) {
+                throw new RuntimeException(e.getMessage());
+            }
+        }
+    }
+
     private List<File> generateRestClientFiles() throws URISyntaxException {
         OpenApiClientGeneratorWrapper generatorWrapper = createGeneratorWrapper("simple-openapi.json").withCircuitBreakerConfig(
                 Map.of("org.openapitools.client.api.DefaultApi", List.of("opThatDoesNotExist", "byeMethodGet")));

--- a/client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java
+++ b/client/deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java
@@ -9,11 +9,14 @@ import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.net.URI;
 import java.net.URISyntaxException;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
@@ -760,15 +763,34 @@ public class OpenApiClientGeneratorWrapperTest {
     }
 
     private Path getOpenApiSpecPath(String specFileName) throws URISyntaxException {
-        return Path.of(requireNonNull(this.getClass().getResource(String.format("/openapi/%s", specFileName))).toURI());
+        URL url = this.getClass().getResource("/openapi/" + specFileName);
+        Objects.requireNonNull(url, "Could not find /openapi/" + specFileName);
+
+        URI uri;
+        try {
+            uri = url.toURI();
+        } catch (URISyntaxException e) {
+            // this should never happen for a well-formed file URL
+            throw new RuntimeException("Invalid URI for " + url, e);
+        }
+
+        return Paths.get(uri);
     }
 
     private Path getOpenApiTargetPath(Path openApiSpec) throws URISyntaxException {
         return Paths.get(getTargetDir(), "openapi-gen");
     }
 
-    private String getTargetDir() throws URISyntaxException {
-        return Paths.get(requireNonNull(getClass().getResource("/")).toURI()).getParent().toString();
+    private String getTargetDir() {
+        URL url = requireNonNull(getClass().getResource("/"),
+                "Could not locate classpath root");
+        try {
+            return Paths.get(url.toURI())
+                    .getParent()
+                    .toString();
+        } catch (URISyntaxException e) {
+            throw new RuntimeException("Invalid URI for " + url, e);
+        }
     }
 
     private Optional<VariableDeclarator> findVariableByName(List<FieldDeclaration> fields, String name) {

--- a/client/deployment/src/test/resources/openapi/issue-1185.json
+++ b/client/deployment/src/test/resources/openapi/issue-1185.json
@@ -1,0 +1,89 @@
+{
+  "openapi": "3.1.0",
+  "info": {
+    "title": "Docling Serve – PictureDescriptionLocal",
+    "version": "0.13.0"
+  },
+  "paths": {
+    "/v1alpha/convert/file": {
+      "post": {
+        "summary": "Process File with Local Picture Description",
+        "operationId": "processFileWithPictureDescriptionLocal",
+        "tags": ["Docling"],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "multipart/form-data": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "files": {
+                    "type": "array",
+                    "items": {
+                      "type": "string",
+                      "format": "binary"
+                    }
+                  },
+                  "picture_description_local": {
+                    "$ref": "#/components/schemas/PictureDescriptionLocal"
+                  }
+                },
+                "required": ["files", "picture_description_local"]
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "PictureDescriptionLocal": {
+        "type": "object",
+        "title": "PictureDescriptionLocal",
+        "description": "Options for running a local vision-language model in the picture description. Parameters refer to a model hosted on Hugging Face.",
+        "properties": {
+          "repo_id": {
+            "type": "string",
+            "title": "Repo Id",
+            "description": "Repository ID on the Hugging Face Hub."
+          },
+          "prompt": {
+            "type": "string",
+            "title": "Prompt",
+            "description": "Prompt used when calling the vision-language model.",
+            "default": "Describe this image in a few sentences."
+          },
+          "generation_config": {
+            "type": "object",
+            "title": "Generation Config",
+            "description": "Config from Transformers’ GenerationConfig (e.g. max_new_tokens, do_sample).",
+            "default": {
+              "max_new_tokens": 200,
+              "do_sample": false
+            },
+            "examples": [
+              {
+                "do_sample": false,
+                "max_new_tokens": 200
+              }
+            ]
+          }
+        },
+        "required": ["repo_id"]
+      }
+    }
+  }
+}

--- a/docs/modules/ROOT/pages/includes/quarkus-openapi-generator-moqu-wiremock.adoc
+++ b/docs/modules/ROOT/pages/includes/quarkus-openapi-generator-moqu-wiremock.adoc
@@ -15,7 +15,7 @@ endif::add-copy-button-to-config-props[]
 
 [.description]
 --
-Path to the Moqu (relative to the project).
+Path to the Moqu OpenAPI files, relative to the `src/main/resources` directory.
 
 
 ifdef::add-copy-button-to-env-var[]

--- a/docs/modules/ROOT/pages/includes/quarkus-openapi-generator-moqu-wiremock_quarkus.openapi-generator.adoc
+++ b/docs/modules/ROOT/pages/includes/quarkus-openapi-generator-moqu-wiremock_quarkus.openapi-generator.adoc
@@ -15,7 +15,7 @@ endif::add-copy-button-to-config-props[]
 
 [.description]
 --
-Path to the Moqu (relative to the project).
+Path to the Moqu OpenAPI files, relative to the `src/main/resources` directory.
 
 
 ifdef::add-copy-button-to-env-var[]

